README.md

```diff
@@ -23,18 +21,9 @@ Features
 
 The IBM CICS collection includes
 [modules](https://github.com/ansible-collections/ibm_zos_cics/tree/master/plugins/modules/),
-[sample playbooks](https://github.com/ansible-collections/ibm_zos_cics/tree/master/playbooks/),
+[sample playbooks](https://github.com/IBM/z_ansible_collections_samples),
```

Comment:
We might mention that it doesn't actually "include" the sample playbooks any longer, and we could link to where the cics samples are in the sample repo, but we can update that later once it's finalized.

Reply:
@stewartfrancis Sounds good to me, as long as the README allows that customization~